error C1083: Cannot open include file: 'opencv2/img_hash.hpp': No such file or directory

**Describe the bug**
Flutter app for Windows fails to compile. 

The error thrown is: 
``` 
C:\Users\akaita\AppData\Local\Pub\Cache\hosted\pub.dev\dartcv4-2.2.1+1\src\dartcv\contrib\img_hash.h(13,10): error C1083: Cannot open include file: 'opencv2/img_hash.hpp': No such file or directory [C:\Users\akaita\Documents\my_app\.dart_tool\hook [C:\Users\akaita\Documents\my_app\build\windows\x64\flutter\flutter_assemble.vcxproj]
Target dart_build failed : error : Building native assets failed. See the logs for more details. [C:\Users\akaita\Documents\my_app\build\windows\x64\flutter\flutter_assemble.vcxproj]
C:\Program Files\Microsoft Visual Studio\18\Community\MSBuild\Microsoft\VC\v180\Microsoft.CppCommon.targets(254,5): error MSB8066: Custom build for 'C:\Users\akaita\Documents\my_app\build\windows\x64\CMakeFiles\bc8a1b05a3781d9898e30144bd8405cf\flutter_windows.dll.rule;C:\Users\akaita\Documents\my_app\build\windows\x64\CMakeFiles\9432d22eb4d9f445dbc1ef76a344c3c8\flutter_assemble.rule' exited with code 1. [C:\Users\akaita\Documents\my_app\build\windows\x64\flutter\flutter_assemble.vcxproj]
Error: Build process failed.
```

**To Reproduce**

// pub.yaml
``` yaml
dependencies:
  flutter:
    opencv_dart: ^2.2.1+1

hooks:
  user_defines:
    dartcv4:
      # debug: true
      include_modules: # `core` is always included
        - imgproc
        - imgcodecs
        - img_hash
```

Try to build.

**Expected behavior**
I would expect it to build, including the `img_hash` module (which is excluded by default)

**Desktop:**
 - Flutter version 3.38.9
 - OS: Windows 10.0.19045.6466 (10 Pro 64-bit, 22H2)
 - Visual Studio Community 2026 version 18.1.11312.151
 - Windows 10 SDK version 10.0.26100.0

--- a/packages/dartcv/CHANGELOG.md
+++ b/packages/dartcv/CHANGELOG.md
@@ -1,5 +1,9 @@
 # dartcv
 
+## 2.2.1+3
+
+- fix: opencv-contrib modules are not correctly built when changing include modules.
+
 ## 2.2.1+2
 
 - fix: Explicitly disable building unused modules to reduce build time.

--- a/packages/dartcv/pubspec.yaml
+++ b/packages/dartcv/pubspec.yaml
@@ -1,6 +1,6 @@
 name: dartcv4
 description: "OpenCV4 bindings for Dart language and Flutter, using dart:ffi. The most complete OpenCV bindings for Dart!"
-version: 2.2.1+2
+version: 2.2.1+3
 opencv_version: 4.13.0
 homepage: https://github.com/rainyl/opencv_dart
 

--- a/packages/dartcv/src/CMakeLists.txt
+++ b/packages/dartcv/src/CMakeLists.txt
@@ -114,7 +114,7 @@ if(DARTCV_BUILD_OPENCV_FROM_SOURCE)
       URL https://github.com/opencv/opencv_contrib/archive/refs/tags/${OPENCV_VERSION}.tar.gz
     )
     FetchContent_MakeAvailable(opencv_contrib)
-    set(OPENCV_EXTRA_MODULES_PATH ${opencv_contrib_SOURCE_DIR}/modules CACHE PATH "Path to opencv_contrib modules")
+    set(OPENCV_EXTRA_MODULES_PATH ${opencv_contrib_SOURCE_DIR}/modules CACHE PATH "Path to opencv_contrib modules" FORCE)
   endif()
 
   FetchContent_Declare(

--- a/packages/opencv_dart/CHANGELOG.md
+++ b/packages/opencv_dart/CHANGELOG.md
@@ -1,5 +1,9 @@
 # opencv_dart
 
+## 2.2.1+3
+
+- fix: opencv-contrib modules are not correctly built when changing include modules.
+
 ## 2.2.1+2
 
 - fix: Explicitly disable building unused modules to reduce build time.

--- a/packages/opencv_dart/pubspec.yaml
+++ b/packages/opencv_dart/pubspec.yaml
@@ -1,7 +1,7 @@
 name: opencv_dart
 description: |
   OpenCV4 bindings for Flutter, using dart:ffi.
-version: 2.2.1+2
+version: 2.2.1+3
 opencv_version: 4.13.0
 repository: https://github.com/rainyl/opencv_dart
 homepage: https://github.com/rainyl/opencv_dart/tree/main/packages/opencv_dart

--- a/packages/opencv_dart/pubspec.yaml
+++ b/packages/opencv_dart/pubspec.yaml
@@ -13,7 +13,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  dartcv4: ^2.2.1+2
+  dartcv4: ^2.2.1+3
   # dartcv4:
   #   path: ../dartcv
 

